src/rpc/rawtransaction.cpp

```diff
@@ -890,13 +892,15 @@ static RPCHelpMan testmempoolaccept()
                 },
                 RPCResult{
                     RPCResult::Type::ARR, "", "The result of the mempool acceptance test for each raw transaction in the input array.\n"
-                        "Length is exactly one for now.",
+                        "Returns results for each transaction in the same order they were passed in.\n"
+                        "It is possible for transactions to not be fully validated ('allowed' unset) if an earlier transaction failed.\n",
```

Comment:
I thought this RPC was supposed to generally fail if any transactions fail.

Reply:
I changed my mind on it because it seemed like the 1-if-failure API was confusing and less helpful